### PR TITLE
Shop advanced

### PR DESCRIPTION
**Pull request by oterral**
[Test](https://mf-geoadmin3.dev.bgdi.ch/teo_shop/index.html?lang=de&topic=ech&bgLayer=ch.swisstopo.pixelkarte-farbe&X=167335.77&Y=626632.76&dev3d=true&zoom=3&debug&layers=ch.swisstopo.swissimage-product,ch.swisstopo.pixelkarte-farbe-pk25.noscale,ch.swisstopo.geologie-geocover,ch.swisstopo.wk25-papierkarte-zusammensetzung.metadata&layers_opacity=1,1,0.75,1)

Waiting translations

### CODE REVIEW

File: src/components/map/MapService.js [hunk 1]
```diff
@@ -495,6 +510,20 @@ goog.require('ga_urlutils_service');
...
+                'ch.swisstopo.pixelkarte-farbe-pk200.noscale',
+                'ch.swisstopo.swissimage-product',
+                'ch.swisstopo.swissbuildingds3d'
+              ];
+              shop.forEach(function(id) {
+                if (response.data[id]) {
+                  response.data[id].tooltip = true;
+                  response.data[id].available = true;
+                }
+              });
```
Comment (gjn) [inline src/components/map/MapService.js]: @AFoletti @rfrancel Can we put this in bod so we can remove it from front-end code? 

Please do so and deploy bod to dev.

Comment (rfrancel) [inline src/components/map/MapService.js]: the tooltips are set to true in the bod. Where does the available come from @AFoletti?

**Comment by AFoletti** [comment]
Sry @rfrancel , I do not understand your question...

File: rc_user [hunk 2]
```diff
@@ -2,3 +2,4 @@ export USERNAME='$(shell whoami)'
 export API_URL=//mf-chsdi3.dev.bgdi.ch/$USERNAME
 export APACHE_BASE_PATH=/$USERNAME
 export E2E_TARGETURL=https://mf-geoadmin3.dev.bgdi.ch/$USERNAME
+export SHOP_URL=//shop-shopbgdi.openshift.puzzle.ch
```
Comment (gjn) [inline rc_user]: We need this url in rc_dev, rc_int and rc_prod (to the corresponding shop urls).

File: src/components/map/MapService.js (hunk 1 above)
Comment (rfrancel) [inline src/components/map/MapService.js]: bod is deployed. 'available' comes from the table directly. I will deploy the stopo table when I'm done with the views and tell you when it's done.

Comment (gjn) [inline src/components/map/MapService.js]: +1. Thanks alot @rfrancel

File: rc_user (hunk 2 above)
Comment (oterral) [inline rc_user]: The url are already set in rc_file, the problem is for dev `shop-bgdi.dev.bgdi.ch`doesn't work. SHould we use //shop-shopbgdi.openshift.puzzle.ch as dev url?

**Comment by oterral** [comment]
@rfrancel I've re-added the button for papierkarte? can you remove it in the tootip (and the price too)  thx.

https://mf-geoadmin3.dev.bgdi.ch/ltteo/src/index.html?lang=de&topic=ech&bgLayer=ch.swisstopo.pixelkarte-farbe&X=161540.52&Y=573424.73&dev3d=true&zoom=0&debug&layers=ch.swisstopo.lk25-papierkarte.metadata

File: rc_user (hunk 2 above)
Comment (gjn) [inline rc_user]: yes, openshift is official dev url, I think.

**Comment by gjn** [comment]
Not all shop layers are working correctly. Price does sometimes return 404.

We'll attack this tomorrow.

**Comment by gjn** [comment]
I think for standard products like pixelkarte noscale, we have to put 'mapnumber' as parameter to dispatcher/price services to be correct. This needs verification.

**Comment by gjn** [comment]
I've adapted/force pushed the urls in the rc_branch.mako. I'll merge now and deploy.

**Pull request merged**